src/Resources/config/services.xml

```diff
@@ -230,10 +231,11 @@
             id="Sylius\PayPalPlugin\Enabler\PaymentMethodEnablerInterface"
             class="Sylius\PayPalPlugin\Enabler\PayPalPaymentMethodEnabler"
         >
-            <argument type="service" id="sylius.http_client" />
```

Comment:
This change is probably not needed as the `sylius.http_client` is an alias to `Psr\Http\Client\ClientInterface` 🤔

Reply:
change reverted